### PR TITLE
git-ls-{files,remote,tree}: add Korean translation

### PR DESCRIPTION
**Pull request by Zamoca42**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/git-ls-files.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`git ls-files --modified`
+
+- .gitignore에 명시된 파일과 Git이 관리하지 않는 파일 보기:
+
+`git ls-files --others`
+
+- Git이 관리하지 않는 파일 중 .gitignore에 명시되지 않은 파일 보기:
+
+`git ls-files --others --exclude-standard`
```
Comment (Zamoca42) [inline pages.ko/common/git-ls-files.md:20]: `ignored` -> `.gitignore에 명시된 파일`
`untracked` -> `Git이 관리하지 않는 파일`

File: pages.ko/common/git-ls-remote.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- 기본 원격 저장소의 태그 정보만 보기:
+
+`git ls-remote --tags`
+
+- 이름이나 URL을 기반으로 특정 원격 저장소의 모든 브랜치와 태그 정보 보기:
+
+`git ls-remote {{저장소_URL}}`
+
+- 특정 검색어와 일치하는 정보만 보기:
```
Comment (Zamoca42) [inline pages.ko/common/git-ls-remote.md:23]: `pattern` -> `특정 검색어`

File: pages.ko/common/git-ls-tree.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# git ls-tree
+
+> 트리 객체의 파일과 디렉토리 목록을 보여줍니다.
```
Comment (Zamoca42) [inline pages.ko/common/git-ls-tree.md:3]: `contents of the tree` -> `트리 객체의 파일과 디렉토리 목록`

**Review by spageektti — APPROVED** [review]

**Review by IMHOJEONG — APPROVED** [review]
The translation looks good with detailed explanations.

**Pull request merged**